### ISSUE TITLE
Explain option splits

### ISSUE THREAD

**Issue by slinkp** [jlevy/og-equity-compensation#43]
When my company had a series B, our existing options were split - I believe this is done to make a larger pool of shares? Don't really know.  The total of exercise price times number of options stayed the same, it was just more shares at a lower price.

It would be great to say something about how such splits work - why they happen, and how they can affect the employee.  When it happened here, I remember talking to some people about it and we all basically ignored it, thinking that the total product was the same, therefore nothing had really changed. We were thinking about it like this:

```
              Shares  Exercise price  Total exercise cost
pre-split      1,000           $1.00                $1000
post-split    10,000           $0.10                $1000
```

But at  the same time, while the FMV per share went down, it didn't go down by anywhere near the same ratio - the total valuation of the company was higher.  That sounds great - we have shares in a more valuable company, yay! Except that after talking to a tax professional, I found out that the theoretical net gain, of (FMV - exercise) * shares, is used for AMT calculation.  So the cost of exercising the day after the series B was a lot higher than the day before the series B.  Ouch.
Something like this:

```
              Shares  Exercise price  Total cost  FMV/share   Net gain for tax calculation
pre-split      1,000           $1.00       $1000      $2.00                         $1,000
post-split    10,000           $0.10       $1000      $1.00                         $9,000
```

**Comment by CertafydeNoQuantum84** [comment]
I believe this scale shouldn't be Portrayed in such a low amount because of inflation .Shares are based off initial startup capital at a specific time in the Data Location in History timeline . Mostly issuue is with political economic development for digital age and legallity  operation . Ethically challenged tactics of coding in reverse Engineering for Monitoring and Free speech rights violations thereof. Remember as whitehat transparent manners go along with truely  acquiring the intended data and accurate data. For partial Governing over in company investment  packages offered to tech and non investment employees.

**Comment by CertafydeNoQuantum84** [comment]
Taxation. Increased over Data Location Storage Maintenance and Retro-Mod  so wealth can be redistributed Fairly so it wouldn't collapse the customers Market from Knockoff  Security evolved in the betaR&D run

**Comment by jlevy** [comment]
Good suggestion. Thanks for sharing. Open to contributions here!

**Comment by jlevy** [comment]
Topics we should cover would include
- definition of a stock split and common reasons for it
- how this affects existing option and stock holders
- other issues to be aware of with stock splits

If I understand your scenario correctly, however, it wasn't really the stock split that surprised you -- it was just the usual fact that increases in FMV can trigger higher taxes at time of exercise. (Which I hope is already covered clearly.)

**Comment by slinkp** [comment]
Yes, I think I was conflating two things there. The split per se did not affect the tax calculation. It's just that it coincided with an adjustment of FMV.
